src/utils/logging.ts

```diff
@@ -23,7 +23,10 @@ interface LoggerExtension {
 function extendLogger ({ logger, name, enabled, log }: LoggerExtension): Debugger {
   const childLogger: Debugger = logger.extend(name);
   childLogger.log = log || logger.log;
-  childLogger.enabled = enabled || logger.enabled;
+  if (enabled || logger.enabled) {
```

Comment:
The old approach of explicitly enabling logging on a single logger broke with the upgraded `debug` package